ai_review/core/model_adapter.py

```
@@ -0,0 +1,28 @@
+import openai
+from configparser import ConfigParser
```

Ruff (0.8.2)

`configparser.ConfigParser` imported but unused

Remove unused import: `configparser.ConfigParser`

(F401)